src/main/java/org/breedinginsight/brapps/importer/services/processors/ExperimentProcessor.java

```diff
@@ -319,7 +358,16 @@ private void getNewBrapiData(List<BrAPIImport> importRows, List<Column<?>> pheno
             this.observationUnitByNameNoScope.put(key, obsUnitPIO);
 
             for (Column<?> column : phenotypeCols) {
-                PendingImportObject<BrAPIObservation> obsPIO = createObservationPIO(importRow, column.name(), column.getString(i));
+                //If associated timestamp column, add
+                String dateTimeValue = null;
+                if (timeStampColByPheno.get(column.name())!=null) {
```

Comment:
from a code style standpoint, could you put spaces around `!=`.  It will make the code more readable.

Reply:
Fixed